apps/crn-server/src/controllers/research-outputs.ts

```diff
@@ -139,6 +141,13 @@ export default class ResearchOutputs implements ResearchOutputController {
       );
     }
 
+    if (
+      currentResearchOutput.published &&
+      !researchOutputUpdateData.published
+    ) {
+      throw Boom.badRequest();
```

Comment:
some message would be nice, something along the lines of 'cannot unpublish a research output' or whatever